README_LINUX.md

```diff
@@ -108,7 +108,23 @@ Unfortunately, the Qt installer does not allow you to deselect the multi-gigabyt
 Building
 --------
 
-### Step 1: Make a build directory
+### Step 1: Obtain the source code
+
+SuperCollider is hosted on Github: https://github.com/SuperCollider/SuperCollider
+
+Obtaining the SuperCollider source code can be done either by downloading a release tarball, or by cloning the repository.
+
+SuperCollider releases are available to download here: https://github.com/supercollider/supercollider/releases
+
+Cloning the repository can be done with the following command:
+
+    git clone --recurse-submodules https://github.com/SuperCollider/SuperCollider.git
+
+The `--recurse-submodules` option will clone the repository's submodules which are needed to build SuperCollider. The submodules can also be obtained by navigating to the root of your locally cloned SuperCollider repository and running the following command:
+
+  git submodule update --init
```

Comment:
i believe you also want --recursive here for ableton link. and please indent this with 4 spaces to match the rest of the added text

Reply:
thanks! I had no idea `--recursive` was needed here.